src/Components/Aspire.Azure.Messaging.EventHubs/EventHubsComponent.cs

```diff
@@ -101,4 +103,47 @@ protected static string GetNamespaceFromSettings(AzureMessagingEventHubsSettings
                 $"the '{configurationSectionName}' configuration section.");
         }
     }
+    protected sealed class AzureEventHubHealthCheck : IHealthCheck, IAsyncDisposable
```

Comment:
nit, newline above

actually also src/Components/** don't seem to be completely consistent but in general seem to prefer only one non-nested class per file. Eg `SeqHealthCheck` is its own file, `NatsHealthCheck` etc.

Reply:
I get you, but it's only used in the derivations of EventHubsComponent itself, so I find private-protected seems to be the right visibility in this case.